.golangci.yml

```diff
@@ -40,11 +40,6 @@ run:
 
 issues:
   exclude-rules:
-    - path: woodpecker-go/woodpecker/client.go|server/swagger/swagger.go
-      linters:
-        - deadcode
-        - unused
-    # gin force us to use string as context key
```

Comment:
please dont remove unrelated comments :laughing:

Reply:
Good catch.
And oops, this was un-intentionally. 
Will bring it back in.